src/cswinrt/strings/additions/Windows.Storage.Streams/WindowsRuntimeBuffer.cs

```diff
@@ -72,14 +72,15 @@ public static IBuffer Create(byte[] data, int offset, int length, int capacity)
         [ThreadStatic]
         private static IMarshal t_winRtMarshalProxy = null;
 
-        private static void EnsureHasMarshalProxy()
+        private static unsafe void EnsureHasMarshalProxy()
         {
             if (t_winRtMarshalProxy != null)
                 return;
 
             try
             {
-                int hr = RoGetBufferMarshaler(out IntPtr proxyPtr);
+                IntPtr proxyPtr = default;
+                int hr = RoGetBufferMarshaler(&proxyPtr);
                 IMarshal proxy = new ABI.Com.IMarshal(ObjectReference<ABI.Com.IMarshal.Vftbl>.Attach(ref proxyPtr));
```

Comment:
Unrelated, but how come we're using a vtable type here? Should we fix this separately?

Reply:
I noticed that too, I plan on addressing this separately.  But it also not a managed type, so it still AOT compatible.